src/send/mod.rs

```diff
@@ -0,0 +1,99 @@
+#[cfg(test)]
+mod test;
+
+use rocket::{
+  data::{self, FromData},
+  http::Status,
+  Data,
+  Outcome,
+  Request,
+};
+use rocket_contrib::{Json, Value};
+use std::{borrow::Cow, collections::HashMap};
+use validator::{self, ValidationError};
+
+#[derive(Debug, Deserialize)]
+struct Body
+{
+  text: String,
+  html: Option<String>,
+}
+
+#[derive(Debug, Deserialize)]
+struct Email
+{
+  to: Vec<String>,
```

Comment:
I'm surprised this is a `Vec` rather than a singe `String`, do we sometimes send multiple emails in the `To:` field from auth-server?

Reply:
Nope we don't, it's just me failing at YAGNI. And this will make the validation a bit nicer too fwiw, so win win.